cpu/efm32/cpu.c

```diff
@@ -83,6 +83,10 @@ static void dcdc_init(void)
  */
 static void clk_init(void)
 {
+    /* disable LFRCO comparator chopping and dynamic element matching
```

Comment:
Wrapping it in a `_SILICON_LABS_32B_SERIES_1` ifdef should be sufficient.

I've checked, but this seems to be a Series-1 feature only. Series-0 (e.g. STK3700) doesn't have this, and the newer Series-2 doesn't have this either.

```
#ifdef _SILICON_LABS_32B_SERIES_1
    /* disable LFRCO comparator chopping and dynamic element matching
     * else LFRCO has too much jitter for LEUART > 1800 baud */
    CMU->LFRCOCTRL &= ~(CMU_LFRCOCTRL_ENCHOP | CMU_LFRCOCTRL_ENDEM);
#endif
```

Reply:
> the newer Series-2 doesn't have this either.

I bet because it's damn near useless and they saw little reason to keep it around, but I digress...